catalogue_pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraProductionTest.scala

```diff
@@ -161,6 +161,8 @@ class SierraProductionTest
 
       val expectedProductions = List(
         ProductionEvent(
+          label =
+            "LondonArts Council of Great Britain1976TwickenhamCTD Printers1974",
```

Comment:
I'm sure this is right, but it looks wrong.

Reply:
It’s because we don’t have any punctuation in the test data – want me to change?

Reply:
change the test data?  Yes if that makes the expectation more realistic.